Makefile

```diff
@@ -167,6 +172,21 @@ docker-build-storageInitializer:
 docker-push-storageInitializer: docker-build-storageInitializer
 	docker push ${KO_DOCKER_REPO}/${STORAGE_INIT_IMG}
 
+controller-gen:
+ifeq (, $(shell which controller-gen))
```

Comment:
Isn't this covered by users installing dependencies?

Reply:
this isn't part of the dependency in `go.mod` and used anywhere in controller go code, it is only used when running `make manifests`

Reply:
cleaned up a bit with unnecessary code